### DISCUSSION TITLE
How to add Google Adsense to my website?

### DISCUSSION THREAD

**Discussion by sztabas** [datenstrom/community#436] [Ask a question] [answered]
Hello, how to put google ads on websites, scripts from adsense. Thank you in advance

**Comment by schulle4u** [comment, marked as answer]
I can't say anything special about Google Adsense scripts, but in general there are several options available to include the necessary code. I'm assuming JavaScript. 

1. If markdown safe mode is disabled (default), you can put the necessary code in your content files. Use [shared content](https://extensions.datenstrom.se/help/how-to-make-a-website) to automatically load the scripts on each page. 
2. Put the scripts directly into the [HTML layout files](https://extensions.datenstrom.se/help/html-files). 
3. If you want to leave everything untouched, you can create a Google Adsense extension. Use the [example extension](https://github.com/schulle4u/yellow-extension-example) as a starting point. 

If you have special requirements, feel free to post a code example.
